phalcon/assets/manager.zep

```diff
@@ -763,15 +783,23 @@ class Manager
 	 */
 	public function outputCss(string collectionName = null) -> string
 	{
-		var collection;
+		var collection, dependencyInjector, tag, callback;
 
 		if !collectionName {
 			let collection = this->getCss();
 		} else {
 			let collection = this->get(collectionName);
 		}
 
-		return this->output(collection, ["Phalcon\\Tag", "stylesheetLink"], "css");
+		let callback = ["Phalcon\\Tag", "stylesheetLink"];
+
+		let dependencyInjector = this->_dependencyInjector;
+		if typeof dependencyInjector == "object" && dependencyInjector->has("tag") {
+			let tag = dependencyInjector->getShared("tag");
```

Comment:
Can we add a check to see if the `tag` was in the DI container and valid and throw an exception if need be?

Reply:
The `dependencyInjector->has("tag")` already here, on line 797. But why we need throw an exception? It will work w/o 'tag'

Reply:
Never mind.